madeofhuman/maintenance-react
# #160900667 implement authentication

#### What does this PR do?
- implement signup
- implement log in
- write tests

#### Description of Task to be completed?
- implement signup
- implement log in
- write tests

#### How should this be manually tested?
- visit https://maintenance-react-staging.herokuapp.com to see the landing page
- click the sign up or log in buttons to sign up for a new account or log in

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160900667](https://www.pivotaltracker.com/story/show/160900667)
[#160900738](https://www.pivotaltracker.com/story/show/160900738)

#### Screenshots
<img width="1698" alt="screen shot 2018-10-03 at 5 04 33 pm" src="https://user-images.githubusercontent.com/7345686/46426703-4637f080-c737-11e8-9474-70448de5d1b1.png">
<img width="1698" alt="screen shot 2018-10-03 at 5 04 43 pm" src="https://user-images.githubusercontent.com/7345686/46426705-49cb7780-c737-11e8-819f-fbb269a2427f.png">
<img width="1698" alt="screen shot 2018-10-03 at 6 06 59 pm" src="https://user-images.githubusercontent.com/7345686/46426718-4e902b80-c737-11e8-93ee-fb56c7fd4c12.png">


#### Questions:
N/A

[Delivers #160900667 #160900738]
